src/config/experiments.js

```diff
@@ -0,0 +1,24 @@
+// @ts-check
+/**
+ * @typedef {'newBackoffice'} Experiments
+ */
+
+localStorage.setItem(
+  'experiments',
+  JSON.stringify([
+    ...(process.env.REACT_EXPERIMENTS || '').split(','),
+    ...(
+      new URLSearchParams(window.location.search).get('experiments') || ''
+    ).split(','),
+  ])
+);
```

Comment:
How does this behave when imported by more than one component?

Reply:
As expected :) AFAIK (and tested) the module's code run once and been cached. Couldn't find docs for webpack or create-react-app but I believe it works the same as node - https://nodejs.org/docs/latest/api/modules.html#modules_caching